docs/user/README.md

```diff
@@ -38,25 +38,29 @@ file a bug in [Bugzilla in Data Platform & Tools :: Glean.js][gleanjs-bugs].
 > if you want to use Glean.js and is missing some key Glean feature.
 ## Sections
 
-### [Using Glean](./user/index.html)
+### [User's Guides](./user/adding-glean-to-your-project.html)
```

Comment:
Same nit as above: "User's Guides" just feels a little awkward due to the possesive form of user.

Reply:
Does "User Guides" sound better? :)